dts/arm/nxp/nxp_s32z27x_rtu0_r52.dtsi

```diff
@@ -89,12 +89,50 @@
 		};
 
 		pit0: pit@76150000 {
-			compatible = "nxp,kinetis-pit";
+			compatible = "nxp,pit";
```

Comment:
Document this change to the binding name in the 3.6 migration guide.

Reply:
I have documented the change, is there something more specific you want me to note in regards to the binding?